stellar-dotnet-sdk-test/RevokeSponsorshipTest.cs

```diff
@@ -0,0 +1,441 @@
+        Assert.IsTrue(transactionResult.IsSuccess);
+        Assert.IsInstanceOfType(transactionResult, typeof(TransactionResultSuccess));
+        Assert.AreEqual("0.00001", transactionResult.FeeCharged);
+        var results = (List<OperationResult>)((TransactionResultSuccess)transactionResult).Results;
+        Assert.AreEqual(1, results.Count);
+        Assert.IsInstanceOfType(results[0], typeof(RevokeSponsorshipSuccess));
+        
+        account = await _server.Accounts.Account(_sponsoredId);
+
+        // Remove the data so the next won't fail next run
+        var removeDataOperation =
+            new ManageDataOperation.Builder(DataName, (string?)null).SetSourceAccount(_sponsoredAccount).Build();
+        tx = new TransactionBuilder(account)
+            .AddOperation(removeDataOperation)
+            .Build();
+
+        tx.Sign(_sponsoredAccount);
+        await _server.SubmitTransaction(tx);
+    }
+
+    private async Task RevokeOfferSponsorship(long offerId)
+    {
+        Assert.IsNotNull(offerId);
+
+        var account = await _server.Accounts.Account(_sponsoringId);
+
+        var revokeOperation = new RevokeLedgerEntrySponsorshipOperation
+                .Builder(_sponsoredId, offerId)
+            .Build();
+
+        var tx = new TransactionBuilder(account)
+            .AddOperation(revokeOperation)
+            .Build();
+        tx.Sign(_sponsoringAccount);
+
+        var txResponse = await _server.SubmitTransaction(tx);
+        Assert.IsNotNull(txResponse);
+        Assert.IsTrue(txResponse.IsSuccess());
+        Assert.IsNotNull(txResponse.ResultXdr);
+        var transactionResult = TransactionResult.FromXdrBase64(txResponse.ResultXdr);
+        Assert.IsTrue(transactionResult.IsSuccess);
+        Assert.IsInstanceOfType(transactionResult, typeof(TransactionResultSuccess));
+        Assert.AreEqual("0.00001", transactionResult.FeeCharged);
+        var results = (List<OperationResult>)((TransactionResultSuccess)transactionResult).Results;
+        Assert.AreEqual(1, results.Count);
+        Assert.IsInstanceOfType(results[0], typeof(RevokeSponsorshipSuccess));
+
+        await Task.Delay(1000);
+        // Remove the offer so the test won't fail next run
+        account = await _server.Accounts.Account(_sponsoredId);
+
+        var removeOfferOperation = new ManageSellOfferOperation
+                .Builder(new AssetTypeNative(), _assetA, "0", "1.5", offerId)
+            .Build();
+        
+        // Remove the trustline so the test won't fail the next run
+        var removeTrustOperation =
+            new ChangeTrustOperation.Builder(_assetA, "0").SetSourceAccount(_sponsoredAccount).Build();
+        tx = new TransactionBuilder(account)
+            .AddOperation(removeOfferOperation)
+            .AddOperation(removeTrustOperation)
+            .Build();
+
+        tx.Sign(_sponsoredAccount);
+        await Task.Delay(1000);
```

Comment:
why is this here? We already wait for 1 second on line 180 rght? same with other tests.